Snakefile

```diff
@@ -0,0 +1,69 @@
+
+DATASETS, = glob_wildcards("dataset_configs/{dataset}.yml")
+SUPERVISED, = glob_wildcards("model_configs/supervised/{supervised}.yml")
+UNSUPERVISED, = glob_wildcards("model_configs/unsupervised/{unsupervised}.yml")
+
+wildcard_constraints:
+    # Random seeds should be numbers
+    seed="\d+"
```

Comment:
Does this mean that whenever a variable called `seed` is defined in any rule, it has to be a number or the script will throw an error? Or does it only apply in certain places?

Reply:
That's correct, the wildcarding likes to grab multiple fields at a time. Ensuring the seed is always a number helps reduce the number of rule priority commands I have to write